### ISSUE TITLE
Buttons (Program and Resources)(help and contact us ) project register

### ISSUE THREAD

**Issue by Pallvi12** [praveenscience/LGMSoC-Public-Data#37]
Program and Resources take us to the same page when clicked
help and contact us is same page why to create different icons then
project button not working
register button at first page take us to some invalid page of the same site

**Comment by praveenscience** [comment]
Yes, because we're working on the other pages. Thanks for reporting. Will be working on this for the last week of June.
